bot-components/GitHub_GraphQL.ml

```diff
@@ -155,3 +190,18 @@ module UpdateMilestone =
     }
   }
 |}]
+
+module MergePullRequest =
+[%graphql
+{|
+  mutation mergePullRequest($pr_id: ID!) {
+    mergePullRequest(input: {pullRequestId: $pr_id}) {
```

Comment:
It's important that you also provide the `commitHeadline` (since we're not using GitHub's default), the `commitBody` (this is where we list the reviewers) and it would be better to provide the `mergeMethod` (even if we're using GitHub's default, but that would be more future proof).